src/extension/importer/rest/src/main/java/org/geoserver/importer/rest/ImportController.java

```diff
@@ -81,13 +81,13 @@ public ImportController(Importer importer) {
     }
 
     @GetMapping
-    public ImportWrapper getImports() {
+    public ImportWrapper getImports(@RequestParam(required=false) String expand) {
```

Comment:
Not a bug fix; A cosmetic change.
The expand parameter was implemented by some well-hidden code in ImportJSONWriter (which I discovered, and subsequently documented when making the api docs). This makes it a bit more clear that this parameter exists.